test/torchaudio_unittest/compliance_kaldi_test.py

```diff
@@ -170,15 +170,15 @@ def _compliance_test_helper(self, sound_filepath, filepath_key, expected_num_fil
             output = get_output_fn(sound, args)
 
             self._print_diagnostic(output, kaldi_output)
-            torch.testing.assert_allclose(output, kaldi_output, atol=atol, rtol=rtol)
+            self.assertEqual(output, kaldi_output, atol=atol, rtol=rtol)
 
     def test_mfcc_empty(self):
         # Passing in an empty tensor should result in an error
         self.assertRaises(AssertionError, kaldi.mfcc, torch.empty(0))
 
     def test_resample_waveform(self):
         def get_output_fn(sound, args):
-            output = kaldi.resample_waveform(sound, args[1], args[2])
+            output = kaldi.resample_waveform(sound.to(torch.float32), args[1], args[2])
```

Comment:
Could you explain why it is necessary to add `to(torch.float32)` here?

Reply:
I don't why but `sound` equals to int, so you will get `RuntimeError: expected scalar type Long but found Float`
```
In [1]: sound.dtype
Out[1]: torch.int16
```

To reproduce:
```
sound = torch.randint(-2742, 8204, (1, 8000))
kaldi.resample_waveform(sound, 16000, 1000)
~/python/projects/torchaudio/torchaudio/compliance/kaldi.py in resample_waveform(waveform, orig_freq, new_freq, lowpass_filter_width)
    841     num_wavs, length = waveform.shape
    842     waveform = F.pad(waveform, (width, width + orig_freq))
--> 843     resampled = F.conv1d(waveform[:, None], kernel, stride=orig_freq)
    844     resampled = resampled.transpose(1, 2).reshape(num_wavs, -1)
    845     target_length = int(math.ceil(new_freq * length / orig_freq))

RuntimeError: expected scalar type Long but found Float
```

Is it expected? @mthrok

Reply:
ping @mthrok

Reply:
Hi @discort 

Sorry for the late response. I looked at the code, and it seems that this should have been caught in previous PRs but the combination of forgetting to change `skipIfNoSoxBackend` and the removal of `torchaudio.load_wav` caused this. I think this is fine. Thanks for the followup.